licenserc.toml

```diff
@@ -0,0 +1,23 @@
+# manually check and fix the license declaration
```

Comment:
nit: `.licenserc.toml` will be better to make the top directory clean.

Reply:
https://github.com/korandoru/hawkeye/blob/1396067d7c96b8e870639e92d3b0ee8b6171e676/README.md?plain=1#L26
Here always `licenserc.toml`, same as [greptimedb](https://github.com/GreptimeTeam/greptimedb/blob/main/licenserc.toml)